src/App.css

```diff
@@ -115,3 +116,29 @@ nav ul li a:hover {
 .dropdown-content a:hover {
   background-color: #f1f1f1;
 }
+.modal-background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5); /* Semi-transparent black background */
```

Comment:
You can create variables for these colors and reuse them later.
E.g
create a root selector at the begining of the file and declare variables within it.
```
:root {
   --transparent-background: rgba(0, 0, 0, 0.5);
}

.modal-background {
 background-color: var(--transparent-background);
```

This helps in readability as well as reusability.

@ranjandahal079